tests/run-make/relro-levels/rmake.rs

```diff
@@ -0,0 +1,29 @@
+// This tests the different -Crelro-level values, and makes sure that they work properly.
+
+//@ ignore-cross-compile
```

Comment:
Question: is ignoring cross-compilation load-bearing here? We can try to see if this test fails in cross-compiling CI jobs if you remove this for now.

Reply:
Question: can you try removing this `//@ ignore-cross-compile` and we can run try jobs to check if that is needed.